backend/src/schema/resolvers/postsInGroups.spec.js

```diff
@@ -0,0 +1,1495 @@
+                },
+                {
+                  id: 'post-without-group',
+                  title: 'A post without a group',
+                  content: 'As a new user, I do not belong to a group yet.',
+                },
+              ]),
+            },
+            errors: undefined,
+          })
+        })
+      })
+
+      describe('with pending membership of group', () => {
+        beforeEach(async () => {
+          authenticatedUser = await pendingUser.toJson()
+        })
+
+        it('shows a the post of the public group and the post without group', async () => {
+          const result = await query({ query: profilePagePosts(), variables: {} })
+          expect(result.data.profilePagePosts).toHaveLength(2)
+          expect(result).toMatchObject({
+            data: {
+              profilePagePosts: expect.arrayContaining([
+                {
+                  id: 'post-to-public-group',
+                  title: 'A post to a public group',
+                  content: 'I am posting into a public group as a member of the group',
+                },
+                {
+                  id: 'post-without-group',
+                  title: 'A post without a group',
+                  content: 'As a new user, I do not belong to a group yet.',
+                },
+              ]),
+            },
+            errors: undefined,
+          })
+        })
+      })
+
+      describe('as member of group', () => {
+        beforeEach(async () => {
+          authenticatedUser = await allGroupsUser.toJson()
+        })
+
+        it('shows all posts', async () => {
+          const result = await query({ query: profilePagePosts(), variables: {} })
+          expect(result.data.profilePagePosts).toHaveLength(4)
+          expect(result).toMatchObject({
+            data: {
+              profilePagePosts: expect.arrayContaining([
+                {
+                  id: 'post-to-public-group',
+                  title: 'A post to a public group',
+                  content: 'I am posting into a public group as a member of the group',
+                },
+                {
+                  id: 'post-without-group',
+                  title: 'A post without a group',
+                  content: 'As a new user, I do not belong to a group yet.',
+                },
+                {
+                  id: 'post-to-closed-group',
+                  title: 'A post to a closed group',
+                  content: 'I am posting into a closed group as a member of the group',
+                },
+                {
+                  id: 'post-to-hidden-group',
+                  title: 'A post to a hidden group',
+                  content: 'I am posting into a hidden group as a member of the group',
+                },
+              ]),
+            },
+            errors: undefined,
+          })
+        })
+      })
+
+      describe('searchPosts', () => {
```

Comment:
This is inside `profile page posts` an therefore a depth to deep I guess!?

Reply:
This is the search box

Reply:
I know.
But my question was should this be inside `profile page posts`?

Reply:
You are right. Much better as you suggest